ding_kultunaut_import.module

```diff
@@ -144,6 +144,17 @@ function ding_kultunaut_import_cron() {
     $node->field_ding_event_category[LANGUAGE_NONE]['0']['tid'] = _ding_kultunaut_import_category((string) $item->kategori);
     $node->field_ding_event_date[LANGUAGE_NONE][] = _ding_kultunaut_import_event_date($item->startdato, $item->slutdato, $item->tid);
     $node->og_group_ref[LANGUAGE_NONE][0]['target_id'] = _ding_kultunaut_import_assign_library((string) $item->sted);
+    $node->field_ding_event_ticket_link[LANGUAGE_NONE][0]['url'] = (string) $item->billet;
+
+    if (!empty((string) $item->pris)) {
```

Comment:
@bozeak What does this do?

Reply:
This change adds ticket link to ding_event ticket field, and if we have price in feed - this will be added to event's price field.

Reply:
Yes, I mean the price thing. In the feed it looks like this:
```
<pris>Entré: 225 kr.r</pris>
<pris>Entré: Gratis billetter</pris>
```

But the field is INT - how will that work?